eva-server/src/main/java/uk/ac/ebi/eva/server/ws/VariantWSServerV2.java

```diff
@@ -52,7 +52,7 @@
 import static org.springframework.hateoas.mvc.ControllerLinkBuilder.methodOn;
 
 @RestController
-@RequestMapping(value = "/v2/variants", produces = "application/hal+json")
+@RequestMapping(value = "/v2/variants", produces = "application/json")
```

Comment:
Why are you removing `hal+`? I think we should keep it as `application/hal+json` and also this is a PR for genes so it's better not to make modifications about variants endpoints here

```suggestion
@RequestMapping(value = "/v2/variants", produces = "application/hal+json")
```

Reply:
I think I changed it while testing the `Resources` issue , Corrected